Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -5070,7 +5063,7 @@ private void OnPublishComplete(IAsyncResult result)
             }
 
             int requestCount = GoodPublishRequestCount;
-            var minPublishRequestCount = GetMinPublishRequestCount(false);
+            int minPublishRequestCount = GetMinPublishRequestCount(false);
             if (requestCount < minPublishRequestCount)
             {
                 BeginPublish(OperationTimeout);
```

Comment:
Another option were to catch up with missing publish requests in the loop here. But not always...